build_solution.cmd

```diff
@@ -1,6 +1,6 @@
 @echo off
 setlocal
-call setenv_%1 %2
+call setenv_%1 %~2
 cd Solutions\%3
 msbuild /flp:verbosity=detailed /clp:verbosity=minimal %~4
-endlocal&&exit /B %ERRORLEVEL%
```

Comment:
That line hasn't changed. I've edited the file directly on GitHub. What that icon at the end is telling you is that the file is missing the empty line that Github requires to make file comparisons.